test/pylib/db/writer.py

```diff
@@ -0,0 +1,98 @@
+import sqlite3
```

Comment:
copyright/license blurb.

Does this need an update to install-dependencies.sh?

Reply:
There is no need to update it. The sqlite3 module is a part of the standard python library.

Reply:
Nice